<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI</class>
 <widget class="QMainWindow" name="UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>371</width>
              <height>481</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="tab_diagonal">
             <attribute name="title">
              <string>Диагональный тип</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_circular">
             <attribute name="title">
              <string>Циркулярный тип</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="push_button_apply">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>371</width>
              <height>28</height>
             </size>
            </property>
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>721</width>
            <height>511</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>743</width>
             <height>533</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>743</width>
             <height>533</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>981</width>
          <height>21</height>
         </size>
        </property>
        <property name="font">
         <font>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="open_file"/>
    <addaction name="save_file"/>
    <addaction name="save_file_as"/>
    <addaction name="close_file"/>
    <addaction name="separator"/>
    <addaction name="exit_app"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="save_settings"/>
    <addaction name="open_settings"/>
    <addaction name="calibr_camera"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="zoom_in"/>
    <addaction name="zoom_out"/>
    <addaction name="normal_size"/>
    <addaction name="fil_to_window"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_view"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_file"/>
   <addaction name="save_file"/>
   <addaction name="close_file"/>
   <addaction name="separator"/>
   <addaction name="zoom_in"/>
   <addaction name="zoom_out"/>
   <addaction name="fil_to_window"/>
   <addaction name="separator"/>
   <addaction name="exit_app"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="open_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/img/Open_32x32.png</normaloff>:/menu/img/Open_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="save_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/img/Save_32x32.png</normaloff>:/menu/img/Save_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="save_file_as">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="close_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/img/Close_32x32.png</normaloff>:/menu/img/Close_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="exit_app">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/img/Exit_32x32.png</normaloff>:/menu/img/Exit_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="save_settings">
   <property name="text">
    <string>Сохранить настройки</string>
   </property>
  </action>
  <action name="open_settings">
   <property name="text">
    <string>Загрузить настройки</string>
   </property>
  </action>
  <action name="zoom_in">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/img/Zoom In_32x32.png</normaloff>:/menu/img/Zoom In_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Увеличить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="zoom_out">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/img/Zoom Out_32x32.png</normaloff>:/menu/img/Zoom Out_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Уменьшить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="normal_size">
   <property name="text">
    <string>Нормальный</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="fil_to_window">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/img/Fil_to_window_32x32.png</normaloff>:/menu/img/Fil_to_window_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Вписать в окно</string>
   </property>
  </action>
  <action name="calibr_camera">
   <property name="text">
    <string>Калибровка камеры</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
